### PR TITLE
remove reference to opentelemetry-erlang-api in the README

### LINKED ISSUE #171: Link to archived project in the README
https://github.com/open-telemetry/opentelemetry-erlang/blob/master/README.md#design

The section linked above says `If you are instrumenting a project your application should only depend on the OpenTelemetry API application.` However following that link takes you to an archived project that links back around to the original project.

Unclear what the correct course of action is

### PR DESCRIPTION
**Pull request by tsloughter**
Fixes #171

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,10 @@ Requires OTP 21.3 or above.
 
 The [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification) defines a language library as having 2 components, the API and the SDK. The API must not only define the interfaces of any implementation in that language but also be able to function as a noop implementation of the tracer. The SDK is the default implementation of the API that must be optional.
 
-If you are instrumenting a project your application should only depend on the [OpenTelemetry API](https://github.com/open-telemetry/opentelemetry-erlang-api/) application. 
+If you are instrumenting a project your application should only depend on the
```
Comment (CrowdHailer) [inline README.md:16]: When would I be instrumenting a "project"?

Should this be "if you are instrumenting a project that others will depend upon"?

Comment (tsloughter) [reply README.md:16]: Even if the application being instrumented isn't used by others the user should still stick to the API unless they are writing something that requires direct SDK access, like an exporter.

Comment (CrowdHailer) [reply README.md:16]: Can i suggest "if you are instrumenting" becomes "when instrumenting"

Comment (tsloughter) [reply README.md:16]: Yea, that sounds better. Can you make a github suggestion so I can just merge it in here.

Comment (CrowdHailer) [reply README.md:16]:
```suggestion
when instrumenting a project your application should only depend on the
```

**Review by ferd — APPROVED** [review]

**Pull request merged**